docs/src/app/components/pages/get-started/usage.md

```diff
@@ -27,7 +50,7 @@ import {RaisedButton} from 'material-ui';
 
 This will make your build process faster and your build output smaller.
```

Comment:
With webpack 1, the module after the `from` keyword is the module *imported* in the bundle.
If you import `material-ui`, you are *importing* everything in your bundle.